### PR TITLE
[BZ2106760] Change delete-local-data to delete-emptydir-data

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.10+

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2106760

Link to docs preview:
- http://file.str.redhat.com/~sniemann/delete_emptydir/support/troubleshooting/troubleshooting-crio-issues.html#cleaning-crio-storage
- http://file.str.redhat.com/~sniemann/delete_emptydir/post_installation_configuration/node-tasks.html#rhel-removing-rhcos_post-install-node-tasks
- http://file.str.redhat.com/~sniemann/delete_emptydir/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-evacuating_nodes-nodes-working
- http://file.str.redhat.com/~sniemann/delete_emptydir/virt/node_maintenance/virt-setting-node-maintenance.html

Additional information: Flag --delete-local-data has been deprecated see:
https://access.redhat.com/solutions/6801291

### CODE REVIEW

File: modules/nodes-nodes-working-evacuating.adoc [hunk 1]
```diff
@@ -89,12 +89,12 @@ value of `0` sets an infinite length of time:
 $ oc adm drain <node1> <node2> --timeout=5s
 ----
 
-** Delete pods even if there are pods using emptyDir using the `--delete-local-data` flag set to `true`. Local data is deleted when the node
+** Delete pods even if there are pods using emptyDir using the `--delete-emptydir-data` flag set to `true`. Local data is deleted when the node
```
Comment (pneedle-rh) [inline modules/nodes-nodes-working-evacuating.adoc:92]: Should "emptyDir" be "`emptyDir` volumes"?

Comment (pneedle-rh) [reply modules/nodes-nodes-working-evacuating.adoc:92]: You could remove the duplication of "using" in the first sentence as follows:

> ** Delete pods even if there are pods using `emptyDir` volumes by setting the `--delete-emptydir-data` flag to `true`.

**Review by pneedle-rh — COMMENTED** [review]
@SNiemann15 I have completed my review of this PR. I have added a couple of minor suggestions. Aside from those, the PR looks good to me!

Please let me know if you have any questions. Thanks!

**Comment by tdaleibm** [comment]
Thanks Silke, LGTM!

**Comment by tdaleibm** [comment]
I checked out oc clis from the past versions, looks like starting with 4.7 oc gives the warning that "--delete-local-data has been deprecated " . 4.6 does not give any warnings so I assume we want to update 4.7 docs onward.

**Comment by pneedle-rh** [comment]
@SNiemann15 as discussed, please create a manual cherry pick PR for 4.10 and for any relevant earlier branches. Thanks!

**Comment by bergerhoffer** [comment]
Per conversation with @SNiemann15 on slack, removing the 4.10 label, since she needs to investigate and create separate PRs for backporting after discussing further with QE.

**Pull request merged**